### DISCUSSION TITLE
How to add some links (Facebook, Insta, Tiktok, Youtube) to contact?

### DISCUSSION THREAD

**Discussion by longtth** [monicahq/monica#5999] [Q&A]
Hi, 
I'm trying monica, and can not find out how to add links (Facebook, Insta, Tiktok, Youtube) to contact. 
Is there a feature to add links? How could I do it? 
Regards, LongNX.

**Comment by ragelfundionctrumnar** [comment]
Go to settings (gear icon) / personalization/ contact field types
Click "add new field type"

Then on the contact page under "contact information" if you click "Add" you can add any of those links.
